equipment-register.xsd

```diff
@@ -669,84 +698,85 @@
                 ```
             </xsd:documentation>
         </xsd:annotation>
         <xsd:sequence>
-            <xsd:element name="parse" type="msl:nonEmptyString">
+            <xsd:element name="value">
                 <xsd:annotation>
                     <xsd:documentation>
-                        The equation to parse to calculate the corrected value.
+                        The equation to parse to calculate the corrected `value`.
+                        If the `value` is a constant numerical value (i.e., an offset),
+                        set the `variables` attribute value to be empty (the `variables`
+                        attribute must still be defined).
 
-                        See the Documentation of [equation](#type_equation) for
-                        more details.
-                    </xsd:documentation>
-                </xsd:annotation>
-            </xsd:element>
-            <xsd:element name="variables">
-                <xsd:annotation>
-                    <xsd:documentation>
-                        A comma-separated string of the variable name(s) in
-                        the equation (the variables that represent the measured
-                        quantities).
-
-                        Each variable name must begin with a letter and may
-                        be followed by zero or more alphanumeric characters.
-                        No whitespace shall be used.
-
-                        **Examples**
-                        * &lt;variables&gt;t&lt;/variables&gt;
-                        * &lt;variables&gt;T&lt;/variables&gt;
-                        * &lt;variables&gt;x1,x2,x3&lt;/variables&gt;
+                        See the documentation of [equation](#type_equation) for
+                        more details about the characters that may be used.
+
+                        See the documentation of [variables](#type_variables) for
+                        more details about the `variables` attribute.
                     </xsd:documentation>
                 </xsd:annotation>
-                <xsd:simpleType>
-                    <xsd:restriction base="xsd:string">
-                        <xsd:pattern value="[a-zA-Z]\w*(,[a-zA-Z]\w*)*"/>
-                    </xsd:restriction>
-                </xsd:simpleType>
+                <xsd:complexType>
+                    <xsd:simpleContent>
+                        <xsd:extension base="msl:nonEmptyString">
+                            <xsd:attribute name="variables" type="msl:variables" use="required"/>
+                        </xsd:extension>
+                    </xsd:simpleContent>
+                </xsd:complexType>
             </xsd:element>
             <xsd:element name="uncertainty">
                 <xsd:annotation>
                     <xsd:documentation>
-                        The standard or expanded uncertainty of the corrected value.
+                        The equation to parse to calculate the standard (or expanded)
```

Comment:
I'd like to suggest this equation should be for the standard uncertainty only. I'm thinking it could be confusing later on if this element was used for the expanded uncertainty for some equipment but standard uncertainty for other equipment. The coverage factor is included in the calibration element so it's easy to convert between them in a known way if we specify this uncertainty element is the standard uncertainty.

Reply:
The `<coverageFactor>` element has been removed.